src/reactjs/types.ts

```diff
@@ -21,7 +22,9 @@ export interface UseBaseQueryOptions<
     TData,
     TQueryData,
     TQueryKey
-  > {}
+  > {
+  context?: React.Context<QueryClient | undefined>
```

Comment:
I think it would be better to have this in a shared `ContextOptions` interface that we can extend everywhere

Reply:
Done. Pushed a new commit to address this.